src/CoreWCF.Primitives/src/CoreWCF/IdentityModel/SecurityUtils.cs

```diff
@@ -340,6 +376,16 @@ internal static AuthorizationContext CreateDefaultAuthorizationContext(IList<IAu
 
             return authorizationContext;
         }
+        public static bool IsRequestSecurityContextIssuance(String actionString)
+        {
```

Comment:
Whatever calls this is doing the wrong thing. You have hard coded in the specific spec version. You should be using the configured SecurityVersion class properties to get the Action.